.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - pip install tox-travis
 
 script:
-  - tox
+  - (cd opentelemetry-api; tox)
```

Comment:
Just wondering, is this part of the WIP or are we only building what is inside the api package?

Reply:
We'll eventually build everything, for now only API has code to build.